R/TerminatorCombo.R

```diff
@@ -67,8 +68,48 @@ TerminatorCombo = R6Class("TerminatorCombo",
       super$print(...)
       catf(str_indent("* Terminators:", paste(map_chr(self$terminators, format),
         collapse = ",")))
+    },
+
+    #' @description
+    #' Returns the remaining runtime in seconds. If `any = TRUE`, the remaining
+    #' runtime is determined by the time-based terminator with the shortest time
+    #' remaining. If non-time-based terminators are used and `any = FALSE`,
+    #' the the remaining runtime is always `Inf`.
+    #' @return `integer(1)`.
+    remaining_time = function(archive) {
+      if(self$param_set$values$any) {
```

Comment:
To save some code you have two options:
`minmax = if (...) min else max` and call `minmax(...)` below or introduce a multiplier `mult = -1` or `1` and then call `min(multi * ...) * mult`